src/components/Publish/Pricing/Fees.module.css

```diff
@@ -18,3 +18,9 @@
 .fees input {
   max-width: 5rem;
 }
+
+@media screen and (max-width: 41rem) {
```

Comment:
we do mobile first in our CSS codebase so only work with `min-width` and adapt elements as you scale up viewports. So the default for all elements is their mobile view and as you scale up the viewport they get adapted if needed

Reply:
Sorry for this weird correction. For some reason my local github wasn't able to push to the correct branch. It got stuck on the re-named old branch.